include/boost/beast/core/buffer_traits.hpp

```diff
@@ -12,24 +12,62 @@
 
 #include <boost/beast/core/detail/config.hpp>
 #include <boost/asio/buffer.hpp>
+#include <boost/mp11/function.hpp>
 #include <type_traits>
 
 namespace boost {
 namespace beast {
 
-/** Type alias used to obtain the underlying buffer type of a buffer sequence.
+/** Determine if a list of types satisfy the <em>ConstBufferSequence</em> requirements.
+
+    This metafunction is used to determine if all of the specified types
+    meet the requirements for const buffer sequences. This type alias
```

Comment:
The documentation refers to "this type alias" whereas the user will see `struct ...`

Reply:
The docs are right. "struct..." is a workaround for the XSLT not supporting type aliases yet.